examples/wifi-echo/server/esp32/main/af-main.h

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
Are these files common or intended to be generated in real apps? I suggest to move them to `common` since I did not see esp32 specific codes.

Reply:
These files are common, but can't compile without the `gen` bits (which are app-specific) right now.  See <https://github.com/project-chip/connectedhomeip/issues/1630>.

We can move them to `examples/common` if files in there can include app-specific headers (which is possible if we set up the include paths for the apps correctly).

Once https://github.com/project-chip/connectedhomeip/issues/1630 is fixed, we can move these files somewhere else much more easily, of course.